app/views/health/health/goal/hpcs/_hpc.haml

```diff
@@ -15,31 +15,37 @@
                 %span.icon-cross
       .row
         .col-12
-          %label
+          %label.text-bold
             Goal Description
             (including any specific barrier to achieving goal not previously mentioned)
           .health-careplan__goal-details
             = simple_format goal.goal_details
       .row.mb-2
         .col-sm-4
-          %label Start Date
+          %label.text-bold Start Date
           .health-careplan__goal-start-date
             = goal.start_date
       .row
         .col-sm-6.mb-2
-          %label Recommended Action Steps and any needed accommodations
+          %label.text-bold Recommended Action Steps and any needed accommodations
         .col-sm-6.mb-2
-          %label Timeframe
-      .row
-        .col-sm-6.mb-2
-          .health-careplan__goal-intervention
-            = simple_format goal.intervention
-        .col-sm-6.mb-2
-          .health-careplan__goal-timeframe
-            = simple_format goal.timeframe
+          %label.text-bold Timeframe
+      - (0..9).each do |i|
+        - action_step = goal["action_step_#{i}"]
+        - timeframe = goal["timeframe_#{i}"]
+        - next unless action_step || timeframe
+        .row
+          .col-sm-6.mb-2
+            .health-careplan__goal-intervention
+              - if action_step&.strip&.present?
+                = simple_format "#{i+1}. #{action_step}"
```

Comment:
If the next on line 36 triggers, this will result in non-sequential numbers... Is that the desired behavior?

Reply:
It's not desired, but shouldn't be an issue.